test-clients/src/main/java/com/hedera/services/bdd/suites/contract/traceability/NewTraceabilitySuite.java

```diff
@@ -89,8 +680,35 @@ private HapiApiSpec vanillaBytecodeSidecar() {
                                 .hasKnownStatus(SUCCESS)
                                 .via(firstTxn))
                 .then(
-                        contractBytecodeSidecar(
-                                EMPTY_CONSTRUCTOR_CONTRACT, vanillaBytecodeSidecar, firstTxn));
+                        withOpContext(
+                                (spec, opLog) -> {
+                                    final HapiGetTxnRecord txnRecord = getTxnRecord(firstTxn);
+                                    allRunFor(
+                                            spec,
+                                            txnRecord,
+                                            expectContractActionSidecarFor(
+                                                    firstTxn,
+                                                    List.of(
+                                                            ContractAction.newBuilder()
+                                                                    .setCallType(CREATE)
+                                                                    .setCallOperationType(
+                                                                            CallOperationType
+                                                                                    .OP_CREATE)
+                                                                    .setCallingAccount(
+                                                                            spec.registry()
+                                                                                    .getAccountID(
+                                                                                            GENESIS))
+                                                                    .setRecipientContract(
+                                                                            spec.registry()
+                                                                                    .getContractId(
+                                                                                            EMPTY_CONSTRUCTOR_CONTRACT))
+                                                                    .setGas(197000)
+                                                                    .setGasUsed(66)
```

Comment:
Are we concerned that the intrinsic gas cost isn't reflected in the `gas` and `gasUsed` fields for the top-level actions? So currently the `gas` field is `initialGas-intrinsicGas` and `gasUsed` is `gasUsed - intrinsicGas`

Reply:
This reads right to me.  The gas is deducted before the first operation is executed in the EVM.  The EVM only ever has `initialGas-intrinsicGas`

Reply:
Got it, thanks Danno